doc/elements.md

```diff
@@ -95,6 +95,24 @@ Attribute | Type | Default | Description
 `suffix` | text | — | A suffix to display after the input box (e.g., `suffix="items"`).
 `display` | "block" or "inline" | "inline" | How to display the input field.
 
+## `pl-string-input` element
+
+```html
+<pl-string-input answers_name="x"></pl-string-input>
+```
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`answers_name` | string | — | Variable name to store data in.
+`weight` | integer | 1 | Weight to use when computing a weighted average score over elements.
+`correct_answer` | string | special | Correct answer for grading. Defaults to `data["correct_answers"][answers_name]`.
+`label` | text | — | A prefix to display before the input box (e.g., `label="$x =$"`).
+`suffix` | text | — | A suffix to display after the input box (e.g., `suffix="items"`).
+`display` | "block" or "inline" | "inline" | How to display the input field.
+`remove_leading_trailing` | boolean | False | Whether or not to remove leading and trailing blank spaces from the input string.
+`remove_between_spaces` | boolean | False | Whether or not to remove blank spaces between characters from the input string.
```

Comment:
I think this might be better named `remove_spaces`, because looking at the code it actually removes all spaces, not just spaces between characters?